client/src/main/resources/process-0.1.xsd

```diff
@@ -322,7 +329,7 @@
         <xs:attribute type="xs:string" name="name" use="optional"/>
         <xs:attribute type="xs:string" name="version" use="optional" default="1.0"/>
         <xs:attribute type="engine-type" name="engine" use="optional" default="oozie"/>
-        <xs:attribute type="xs:string" name="path" use="required"/>
+        <xs:attribute type="xs:string" name="path" use="optional"/>
```

Comment:
I have keep it "required" only and will ask user to specify spark application(jar/python) directory path with Process entity workflow path.